crates/runestick/src/function.rs

```diff
@@ -25,6 +25,15 @@ where
     inner: Inner<V>,
 }
 
+/// A callable non-sync raw.
+pub type RawFunction = RawImpl;
```

Comment:
Name `RawImpl` to `RawFunction` directly instead of a type alias? `RawImpl` would leak out due to this alias anyway but the doc might end up being a bit awkward.

Reply:
Yeah, will do. Just followed the default pattern but when the parametrization of Value/ConstValue went away I was a bit torn on whether keeping the pattern was worthwhile.